resources/ext.wikibase.export.js

```diff
@@ -132,6 +132,10 @@ $( function () {
 					widget.statements.toggle( false );
 					widget.statements.setValue( widget.statements.menu.getItems() );
 				} else {
+					// TODO: is there an OOUI way to clear everything?
+					widget.statements.getValue().forEach( function ( statement ) {
+						widget.statements.removeTagByData( statement );
+					} );
```

Comment:
My JS is a bit rusty so I'm not sure you can actually pass a reference to the function directly into the higher-order forEach. Nowadays you can even do this in PHP (8.1+)

Reply:
I mean

```js
widget.statements.getValue().forEach( widget.statements.removeTagByData );
```

Reply:
Doesn't look like it. The closest I can get is
```js
widget.statements.getValue().forEach( statement => widget.statements.removeTagByData( statement ) );

```

Reply:
Let's see how functional my twitter followers end up being https://twitter.com/JeroenDeDauw/status/1591206141928779777 :)